django_plotly_dash/dash_wrapper.py

```diff
@@ -24,32 +24,28 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 '''
+import inspect
 import itertools
 import json
-import inspect
 import warnings
+from typing import Dict, List, Callable
 
 import dash
 from dash import Dash, dependencies
 from dash._utils import split_callback_id, inputs_to_dict
-
-from flask import Flask
-
 from django.urls import reverse
 from django.utils.text import slugify
-
+from flask import Flask
 from plotly.utils import PlotlyJSONEncoder
 
 from .app_name import app_name, main_view_label
 from .middleware import EmbeddedHolder
-
-from .util import static_asset_path
 from .util import serve_locally as serve_locally_setting
 from .util import stateless_app_lookup_hook
+from .util import static_asset_path
 
 try:
     from dataclasses import dataclass
-    from typing import Dict, List
```

Comment:
This should probably stay here (and add Callable also)

Reply:
Or we may just remove the type annotations for now as they are not essential